packages/subproviders/CHANGELOG.md

```diff
@@ -3,6 +3,9 @@
 ## v0.7.0 - _March 8, 2018_
 
     * Updated legerco packages. Removed node-hid package as a dependency and make it an optional dependency. It is still used in integration tests but is causing problems for users on Linux distros. (#437)
+    * Export `GanacheSubprovider` and `Subprovider` (#426)
+    * Make all subproviders to derive from `Subprovider` (#426)
+    * Add types for `NextCallback`, `OnNextCompleted` (#426)
```

Comment:
Thank you! 🙏

Reply:
I think 0.7.0 has already been published, might need to bump